sdk/metric/meter.go

```diff
@@ -436,7 +439,7 @@ func (r observer) ObserveFloat64(o metric.Float64Observable, v float64, opts ...
 		return
 	}
 
-	if _, registered := r.float64[oImpl.observablID]; !registered {
+	if _, registered := r.float64[oImpl.observablID]; !registered && !oImpl.dropAggregation {
```

Comment:
I made a mistake in my last suggestion. The conditional block before did not include the return, now the unified one does. This means that now if an instrument has a drop aggregator this block will be skipped and the return not executed. That is not what we want given we do not want to run the code below.

Sorry for my mistake, can you please revert my last suggestion. :facepalm: